src/components/Beers.js

```diff
@@ -0,0 +1,67 @@
+import {useState, useEffect} from "react"
+import {NavLink} from "react-router-dom"
+import axios from "axios"
+
+export function Beers(props){
+    const {beers, setBeers} = props
+    const [searchTerm, setSearchTerm] = useState("")
+    const [filteredBeers, setFilteredBeers] = useState(props.beers)
+
+    const handleSearchTermChange = ({target}) => {
+        setSearchTerm(target.value)
+    }
+
+    useEffect(() => {
+        axios.get(`${process.env.REACT_APP_URL}/search?q=${searchTerm}`)
+            .then(res => {
+                setFilteredBeers(res.data)
+            })
+            .catch(err => {
+                console.log("An error occurred while querying beers:", err);
+            }, [searchTerm])
```

Comment:
`, [searchTerm]` that piece of code should be on the array dependency for the `useEffect()`, right now it's throwing request to the API one after the other, and that's it's not a good thing (imagine you have a limited free requests that you could make, this could be a problem really fast)